backend/src/sms/middlewares/sms-stats.middleware.ts

```diff
@@ -12,8 +12,9 @@ const getStats = async (
   next: NextFunction
 ): Promise<Response | void> => {
   const { campaignId } = req.params
+  const { refresh } = req.query
   try {
-    const stats = await SmsStatsService.getStats(+campaignId)
+    const stats = await SmsStatsService.getStats(+campaignId, refresh)
```

Comment:
Could we break this into
```javascript
if(refresh) await SmsStatsService.refreshStats(campaignId)
const stats = await SmsStatsService.getStats(campaignId)
```
Do you recall our query param `?reallySend=true`? This alarmingly reminds me of it, because it's not clear immediately what the effect of the `refresh` parameter is.

Reply:
Alright, I can. Although arguably this reeks more of a param / variable naming issue

Reply:
See https://martinfowler.com/bliki/FlagArgument.html